evap/staff/templates/staff_evaluation_textanswers_quick.html

```diff
@@ -200,6 +201,7 @@ <h5 class="modal-title" id="hotkeys-modal-title">Hotkeys</h5>
             document.querySelector("#shared-full-textanswer-flag-form"),
             "{% url 'staff:evaluation_textanswers_skip' %}",
         );
-        slider.attach();
+        const hash = document.location.hash
+        slider.attach(hash.startsWith("#textanswer-") ? hash.substring(12) : null)
```

Comment:
can you move it into a variable so that I don't have to be afraid that the string is truly 12 characters long?